src/components/PostList/PostList.jsx

```diff
@@ -1 +1,11 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
+    <div className="PostList">
+      {posts.map(post => {
+        return <PostInfo key={post.id} post={post} />;
+      })}
+    </div>
+  </>
+);
```

Comment:
This file is exactly the same with PostList/index.js

It is probably some sort of copy error